src/overlays/actors/ovl_En_Ru1/z_en_ru1.c

```diff
@@ -1462,13 +1462,13 @@ void func_80AEDEF4(EnRu1* this, PlayState* play) {
     }
 }
 
-void func_80AEDFF4(EnRu1* this, PlayState* play) {
+void EnRu1_UpdatePosition(EnRu1* this, PlayState* play) {
     func_80AEDB30(this, play);
-    func_80AEDEF4(this, play);
```

Comment:
based on the other function this calls, which isnt documented here, this seems more about "movement" in general. 
This should maybe be named after `func_80AEDB30` is named though?

Reply:
#2598 names this `EnRu1_UpdateBgCheckFlags` (yes, I know that'll result in a merge conflict, I'm OK with that)